pkgs/top-level/python-packages.nix

```diff
@@ -30195,4 +30233,162 @@ in modules // {
     };
   };
 
+  homeassistant = buildPythonApplication rec {
+    name = "${pname}-${version}";
+    pname = "homeassistant";
+    version = "0.30.2";
+
+    disabled = !isPy3k;
+
+    outputs = [ "out" ];
```

Comment:
why do you specify `outputs` when you only have the default output? This can be removed.